engine/server.js

```diff
@@ -104,9 +104,9 @@ class ChannelEngine {
         playheadDiffThreshold: this.streamerOpts.playheadDiffThreshold,
         maxTickInterval: this.streamerOpts.maxTickInterval,
         profile: channel.profile,
-        slateUri: this.defaultSlateUri,
-        slateRepetitions: this.slateRepetitions,
-        slateDuration: this.slateDuration,
+        slateUri: channel.channel.defaultSlateUri ? channel.channel.defaultSlateUri : this.defaultSlateUri,
```

Comment:
Maybe I could propose to have an object `slate` in the Channel class and you would then have:
```
slateUri: channel.slate && channel.slate.uri ? channel.slate.uri : this.defaultSlateUri
```

and in the class Channel we add the get function `slate()` to the interface:

```
get slate() {
   return this.channel.slate;
}
```
Then the Channel class interface is still abstracted to the implementation outside of the Channel Engine.